### ISSUE TITLE
VSCode: Custom keyboard shortcut for Svelte's Emmet expansion does not work

### ISSUE THREAD

**Issue by Ciantic** [sveltejs/language-tools#558]
**Describe the bug**
Editing Svelte document does not use the same keyboard shortcut for Emmet expansion as other files.

**To Reproduce**

Set a custom shortcut for Emmet abbreviation expansion to your shortcuts JSON:

E.g. Ctrl+Enter
```
  {
    "key": "ctrl+enter",
    "command": "editor.emmet.action.expandAbbreviation",
    "when": "editorTextFocus"
  }
```

Then try to use it e.g. in the HTML document, it should work `div.foo` + `CTRL+Enter` becomes `<div class="foo"></div>`.

However if you try this inside svelte document the entire custom shortcut does not work. Svelte only recognizes tabulator which I don't want to use for Emmet at all.

**Expected behavior**

Shortcut should work.

**System (please complete the following information):**
 - OS: Windows 10
 - IDE: VSCode
 - Plugin/Package: Svelte for VSCode 102.0.0

**Comment by jasonlyu123** [comment]
We didn't use the feature of the emmet extension in the vscode. That won't work.

**Comment by Ciantic** [comment]
Okay thanks, I actually suspected that. I tried to find alternative keyboard command name (svelte specific Emmet), but couldn't find any. Is that keyboard shortcut hardcoded as tabulator?

**Comment by dummdidumm** [comment]
I'm not sure if I understand the problem completely. The shortcut should expand the abbreviation after you have typed something but are not in a autocompletion context. If you are in a autocompletion context, tab/enter will do the completion. If you are not in such a context, I cannot expand the abbreviation at all at the moment - which is what we would need to add support for.

**Comment by dummdidumm** [comment]
You could try to treat Svelte as HTML in the emmet settings: `"emmet.includeLanguages": { "svelte": "html" }`. Does this work as expected for you then?

**Comment by brunobely** [comment]
Sharing what I did:

I had `"emmet.includeLanguages": {  "svelte": "javascriptreact" }` set from a previous configuration (I think one or two years ago, and I think it worked back then. Now it doesn't but changing it to `"emmet.includeLanguages": {  "svelte": "html" }` fixes the issue.

**Comment by Ciantic** [comment]
@fxcie that's yet another shortcut, wrap with abbreviation is different command. 

To sum it up, I use this kind of setup on *all* the other files, e.g. TSX/JSX/HTML:

Tab key = auto-complete (TypeScript / JavaScript functions, symbols etc.)
Ctrl enter = expand emmet e.g. `div.foo` => `<div  class="foo"></div>
Ctrl + shift + enter = wrap with abbrevation, 1. select a text and then ctrl+shift+enter

I don't want the emmet to ruin Tab key completions, because emmet abbreviations are shoddy guesses, I like my tab key to do *precisely* symbol auto-completion. This is about muscle memory, if I have to look at the auto-completion list if it's a) emmet abbreviation or b) TS/JS symbol, it slows me down.

**Comment by jasonlyu123** [comment]
As I remembered, at the time when the issue was created. The `"emmet.includeLanguages": { "svelte": "html" }` config doesn't include emmet for CSS inside style tag. But it seems to be fixed in vscode 1.55 back in March. Also, we have added configs to disable emmet completion in CSS and HTML respectively. So you can just disable the emmet included in the svelte extension and use the build-in emmet extension now.

Here're the mentioned configs:
```json
{
    "emmet.includeLanguages": {
        "svelte": "html"
    },
    "svelte.plugin.css.completions.emmet": false,
    "svelte.plugin.html.completions.emmet": false
}
```

**Comment by jasonlyu123** [comment]
Closing due to inactive and have a workaround.
